matching/.gitignore

```diff
@@ -0,0 +1,3 @@
+.stack-work/
+haste.cabal
```

Comment:
I believe this should now be matching.cabal.

Reply:
I see no reason not to make it *.cabal